test/Downloads.jl.faceup

```diff
@@ -41,7 +41,7 @@ resources are not cleaned up until `Downloader` is garbage collected.
     ca_roots::«t:Union»{«t:String», «t:Nothing»}
     easy_hook::«t:Union»{«t:Function», «t:Nothing»}
 
-    «f:Downloader»(multi::«t:Multi») = «k:new»(multi, get_ca_roots(), EASY_HOOK[])
+    «f:Downloader»(multi::«t:Multi») = new(multi, get_ca_roots(), EASY_HOOK[])
```

Comment:
Would be nice if "new" got font-locked as either builtin or keyword here, but I don't think it's worth the spurious fontification when "new" is a regular var.

Reply:
Yeah, I don't think it's possible using tree-sitter alone to recognize places where new is a constructor call.